examples/rl_gsm8k/utils.py

```diff
@@ -69,6 +69,11 @@ def __init__(
         self.stderr_file: Optional[TextIO] = None
         self.stats = {}
 
+        # Add node rank awareness
+        self.node_rank = int(os.environ.get("RANK", 0))
+        self.port_offset = self.node_rank * 1000  # Ensure different port ranges for each node
```

Comment:
why different port ranges for each node?

Reply:
I'm not sure I understand. These vllms are running on different jobs, aren't they?